scripts/ci.requirements.txt

```diff
@@ -10,3 +10,8 @@ pylint == 2.4.4
 # Use the earliest version of mypy that works with our code base.
 # See https://github.com/Mbed-TLS/mbedtls/pull/3953 .
 mypy >= 0.780
+
+# Install cryptography to avoid import-error reported by pylint.
+# What we really need is cryptography >= 35.0.0, which is only
+# available for Python >= 3.6.
+cryptography # >= 35.0.0
```

Comment:
While we are waiting on the request to update the version of pip on windows, we can restrict this to only install on linux and get this merged - we are only running pylint on linux anway.
FreeBSD would fail at this step as  well due to missing a rust toolchain, but an older version of cryptography happens to be installed, satisfying the un-versioned requirement.
```suggestion
cryptography >= 35.0.0; sys_platform == 'linux' and python_version >= '3.6'
cryptography;           sys_platform == 'linux' and python_version <  '3.6'
````